src/assets/HowTo.md

```diff
@@ -0,0 +1,111 @@
+# How to?
```

Comment:
Delete this or the other `HowTo.md` and use one file for both app and documentation